source/docs/software/roborio-info/roborio2-imaging.rst

```diff
@@ -0,0 +1,60 @@
+RoboRIO2 Imaging
+================
+
+.. note:: The imaging instructions for the NI roboRIO 1.0 are :ref:`here <docs/zero-to-robot/step-3/imaging-your-roborio:Imaging your roboRIO>`.
+
+The NI roboRIO 2.0 boots from a microSD card configured with an appropriate boot image containing the NI Linux Real-Time OS, drivers, and libraries specific to FRC. There are two ways to create the microSD card -- with the roboRIO :ref:`using the imaging tool <docs/zero-to-robot/step-3/imaging-your-roborio:Imaging your roboRIO>`, or with a laptop and an SD burner application.
+
+microSD Requirements
+--------------------
+
+The NI roboRIO 2.0 controller supports microSD cards.  It is recommended to use a SDHC or SDXC capacity card, though all types are expected to function correctly.
```

Comment:
```suggestion
The NI roboRIO 2.0 controller supports SDHC and SDXC microSD cards with a recommended minimum capacity of 2GB. 
```
I am not aware of any microSD card that is neither SDHC or SDXC. I do not remember what the recommended minimum is. I just went with 2GB as you are not likely to purchase or find microSD cards at a smaller size.

Reply:
I don't get why your interpretation overrides the advice of the device manufacturer.  You should post on teamforge and discuss the recommendation with them!